src/main/groovy/org/savantbuild/plugin/java/testng/JavaTestNGPlugin.groovy

```diff
@@ -73,6 +75,12 @@ class JavaTestNGPlugin extends BaseGroovyPlugin {
    *   groovyTestNG.test(groups: ["unit", "performance"])
    * </pre>
    *
+   * Supported command line options:
+   *  --onlyFailed    only runs test that failed on previous test run
```

Comment:
While you're in here, if you know, or want to figure it out, when using `--onlyFailed` - it returns a non-zero exit code, so the `sb` command thinks it failed, not sure why.

Reply:
The non-zero exit happens when fusionauth-app attempts to run the isolation group tests and no tests run.  I can submit a PR to fix that (and will make sense to do that after this PR merges)